firecrown/likelihood/gauss_family/gauss_family.py

```diff
@@ -15,28 +15,14 @@
 
 
 class GaussFamily(Likelihood):
-    """A Gaussian log-likelihood with a constant covariance matrix.
-
-    Parameters
-    ----------
-    statistics : list of Statistic
-        A list of the statistics
-
-    Attributes
-    ----------
-    cov : np.ndarray, shape (n, n)
-        The covariance matrix.
-    cholesky : np.ndarray, shape (n, n)
-        The (lower triangular) Cholesky decomposition of the covariance matrix.
-    inv_cov : np.ndarray, shape (n, n)
-        The inverse of the covariance matrix.
-
-    Methods
-    -------
-    compute_loglike : compute the log-likelihood
+    """GaussFamily is an abstract class. It is the base class for all likelihoods
+    based on a chi-squared calculation. It provides an implementation of
+    Likelihood.compute_loglike. Derived classes must implement the abstract method
```

Comment:
Is it not the opposite? GaussFamily implements `compute_chisq` and the derived must implement `compute_loglike` using the value returned by `compute_chisq`?

Reply:
You are correct; I'll fix the comment.